### DISCUSSION TITLE
Fulfillment Inbound API v2024-03-20 transportationOption questions

### DISCUSSION THREAD

**Discussion by njiang** [amzn/selling-partner-api-models#4133] [Q&A]
I'm investigating the latest Inbound API following the documentation. I've created a transportation option, and called listTransportationOption api. I got a paginated list of carriers. Most of our clients just want to use the partnered carrier (UPS). Is there a way to just return partnered carriers?  

Also how do we differentiate SP and LTL in the new flow?  Seems if it's LTL we need to submit freightclass and pallet info when creating transportation options?

I also got the following error when I call the confirmTransportationOption api:
{
    "operationStatus": "FAILED",
    "operationId": "883df4b8-33a0-4602-82d4-13a75606e9d9",
    "operation": "confirmTransportationOptions",
    "operationProblems": [
        {
            "severity": "ERROR",
            "code": "InternalServerError",
            "message": "ERROR: Something went wrong. Please try again later."
        }
    ]
}
I have generated and confirmed delivery windows for the shipments. Not sure what am I missing? And the error msg is not helpful at all.

**Comment by MetaMan2023** [comment]
@njiang Are you still having this issue?   If so, please check out this thread in the "Issues" section and reply...

https://github.com/amzn/selling-partner-api-models/issues/4656
